interface/billing/print_daysheet_report_num1.php

```diff
@@ -190,735 +197,822 @@
  switch ($iter{'user'}) {
     case $iter{'user'} = $user_final_list[0]:
         $us0_user = $iter{'user'};
+        $us0_user_name = $iter{'user_fname'} .  $iter{'user_mname'}  .  $iter{'user_lname'};
         $us0_fee = $us0_fee + $iter{'fee'};
         if (($iter{'ins_code'}) > 0 AND ($iter{'code_type'}) === 'Insurance Payment' ){
          $us0_inspay = $us0_inspay + $iter{'ins_code'};
         }
         $us0_insadj = $us0_insadj + $iter{'ins_adjust_dollar'};
         $us0_patadj = $us0_patadj + $iter{'pat_adjust_dollar'};
-        if (($iter{'pat_code'}) > 0 AND ($iter{'code_type'}) === 'Patient Payment' ){ 
-         $us0_patpay = $us0_patpay + $iter{'pat_code'};         
+        if (($iter{'pat_code'}) > 0 AND ($iter{'code_type'}) === 'Patient Payment' ){
+         $us0_patpay = $us0_patpay + $iter{'pat_code'};
         }
         if (($iter{'ins_code'}) < 0 AND ($iter{'code_type'}) === 'Insurance Payment' ){
-          $us0_insref = $us0_insref + $iter{'ins_code'};        
+          $us0_insref = $us0_insref + $iter{'ins_code'};
         }
-        if (($iter{'pat_code'}) < 0 AND ($iter{'code_type'}) === 'Patient Payment' AND $iter{'paytype'} != 'PCP' ){ 
+        if (($iter{'pat_code'}) < 0 AND ($iter{'code_type'}) === 'Patient Payment' AND $iter{'paytype'} != 'PCP' ){
           $us0_patref = $us0_patref + $iter{'pat_code'};
         }
+
```

Comment:
if ($iter{'paytype'})=='PP'{} here maybe

Another would be use  foreach($case as $arg => $v){